### PR TITLE
refactor: replace some lodash methods with native alternatives

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

**Lodash imports removed:**

- isEmpty
- includes
- concat
- some
- keys
- flatten

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by alexbarnsley** [comment]
@brenopolanski I'm wondering if it's worth adapting this PR to use https://github.com/ArkEcosystem/utils as they're sort of the same thing 🤔

**Comment by brenopolanski** [comment]
@alexbarnsley The initial idea is to remove lodash imports as much as possible and use native functions. I think there is no need for now to add this `ArkEcosystem/utils` package.

File: src/renderer/utils/index.js [hunk 1]
```diff
@@ -58,6 +65,7 @@ const min = nums => {
 const upperFirst = str => `${str.charAt(0).toUpperCase()}${str.slice(1)}`
 
 export {
+  cammelToUpperSnake,
```
Comment (faustbrian) [inline src/renderer/utils/index.js:68]:
```suggestion
  camelToUpperSnake,
```

Comment (brenopolanski) [reply src/renderer/utils/index.js]: Thanks @faustbrian

**Pull request merged**